src/main/java/io/r2dbc/postgresql/codec/EnumCodec.java

```diff
@@ -181,6 +198,27 @@ public CodecRegistrar build() {
             };
         }
 
+        /**
+         * An enumeration of codec registration priorities.
+         */
+        public enum RegistrationPriority {
+            FIRST,
+            LAST
+        }
+
+        private static final class EnumMappingConfig {
```

Comment:
Following the comment earlier, we don't need this type.

Reply:
The class has deleted